primitives/src/locktime/absolute.rs

```diff
@@ -438,10 +451,11 @@ mod tests {
     }
 
     #[test]
-    fn lock_time_from_invalid_hex_should_err() {
-        let hex = "0xzb93";
-        let result = LockTime::from_hex(hex);
-        assert!(result.is_err());
+    fn invalid_should_err() {
+        assert!(LockTime::from_hex("0xzb93").is_err());
+        assert!(LockTime::from_unprefixed_hex("zb93").is_err());
+        assert!(LockTime::from_height(1_653_195_600).is_err()); // Above threshold.
+        assert!(LockTime::from_time(750_000).is_err()); // Below threshold.
```

Comment:
Regarding renames: could've been put to a separate test fn.

Also the test numbers should be one above/one below the threshold.

Reply:
Done